modules/covid_vaccine/app/models/covid_vaccine/v0/expanded_registration_submission.rb

```diff
@@ -19,6 +50,54 @@ class ExpandedRegistrationSubmission < ApplicationRecord
       attr_encrypted :raw_form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
       attr_encrypted :eligibility_info, key: Settings.db_encryption_key, marshal: true,
                                         marshaler: JsonMarshal::Marshaller
+
+      # We want records that have acceptable discharge status, country, state, zip and facility
+      def self.to_csv(separater = '^')
+        CSV.generate(col_sep: separater) do |csv|
+          eligible_us.order('created_at DESC').each do |es|
+            csv << es.send(:csv_row)
+          end
+        end
+      end
+
+      private
+
+      def country_us?
+        raw_form_data[:country_name] == 'USA'
+      end
+
+      def csv_row
+        raw_form_data.values_at('first_name', 'middle_name', 'last_name') +
+          [csv_birth_date] +
+          [raw_form_data['ssn']] +
+          [csv_birth_sex] +
+          [icn] +
+          [csv_address] +
+          raw_form_data.values_at('city', 'state_code', 'zip_code') +
+          [csv_phone] +
+          raw_form_data.values_at('email_address', 'preferred_facility') +
```

Comment:
preferred_facility is going to come to us from the frontend as "vha_648", and at some point we need to do a `delete_prefix('vha_'). I think this would be a good point to do it.

Reply:
k, yeah i wasn't sure about those